pkg/handler/sign/hsm_pkcs1_1_5/handler.go

```diff
@@ -0,0 +1,43 @@
+package hsm_pkcs1_1_5
```

Comment:
Go still doesn't like underscore in names. But I'm not going to die on that hill anymore.